pkg/web/static/js/usersManagement.js

```diff
@@ -54,8 +54,8 @@ document.addEventListener('htmx:afterSettle', (e) => {
     };
 
     // TODO: Should input also be included in the events lists?
-    addEventListeners(passwordInput, 'change keyup paste cut', checkPasswords);
-    addEventListeners(confirmPassword, 'change keyup paste cut', checkPasswords);
+    addEventListeners(passwordInput, 'input', checkPasswords);
```

Comment:
I double checked whether `input` would also be needed here. I determined that adding `input` means that the others aren't required since the input is fired every time a change is made to the input, so this would cover 'keyup' and 'paste'. Since we're using the `password` type for the element, users aren't currently able to `cut`. 

The `change` event on the other hand only fires once a value has been committed.

Reply:
glad you found a better event!